Rubberduck.Parsing/Annotations/VBAParserAnnotationFactory.cs

```diff
@@ -28,18 +28,29 @@ public VBAParserAnnotationFactory()
         public IAnnotation Create(VBAParser.AnnotationContext context, QualifiedSelection qualifiedSelection)
         {
             string annotationName = context.annotationName().GetText();
-            List<string> parameters = new List<string>();
-            var argList = context.annotationArgList();
-            if (argList != null)
+            List<string> parameters = AnnotationParametersFromContext(context);
+            return CreateAnnotation(annotationName, parameters, qualifiedSelection);
+        }
+
+            private static List<string> AnnotationParametersFromContext(VBAParser.AnnotationContext context)
             {
-                parameters.AddRange(argList.annotationArg().Select(arg => arg.GetText()));
+                List<string> parameters = new List<string>();
+                var argList = context.annotationArgList();
+                if (argList != null)
+                {
+                    parameters.AddRange(argList.annotationArg().Select(arg => arg.GetText()));
+                }
+                return parameters;
```

Comment:
Why not make it all a bit shorter and go with:

```
var argList = context.annotationArgList();
return argList?.annotationArg().Select(arg => arg.GetText()).ToList() ?? new List<string>();
```

That is if my null-unwrapping operators and stuffs are syntactically correct